expression/builtin_math_vec.go

```diff
@@ -223,3 +225,42 @@ func (b *builtinRoundDecSig) vecEvalDecimal(input *chunk.Chunk, result *chunk.Co
 func (b *builtinRoundDecSig) vectorized() bool {
 	return true
 }
+
+func (b *builtinPowSig) vecEvalReal(input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	buf1, err := b.bufAllocator.get(types.ETReal, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf1)
+	if err := b.args[0].VecEvalReal(b.ctx, input, buf1); err != nil {
+		return err
+	}
+
+	if err := b.args[1].VecEvalReal(b.ctx, input, result); err != nil {
+		return err
+	}
+
+	x := buf1.Float64s()
+	y := result.Float64s()
+	f64s := result.Float64s()
+	for i := 0; i < n; i++ {
+		if buf1.IsNull(i) || result.IsNull(i) {
```

Comment:
We can use `result.MergeNulls()` to calculate the null bitmap for `result`. It's much faster.